### DISCUSSION TITLE
Can a CMSampleBuffer be modified?🥲

### DISCUSSION THREAD

**Discussion by 1domybest** [HaishinKit/HaishinKit.swift#1489] [Q&A]
Is it possible to modify a CMSampleBuffer received from the camera when live broadcasting with RTMP?

Of course, I am aware that using a third-party library may not guarantee the frame rate. It seems that offscreen rendering works on the broadcaster’s screen, but I am curious if it is possible to compress and transmit the filtered image during broadcasting.

**Comment by shogo4405** [comment]
I do not provide free answers to questions about whether something is possible or not. I can respond for 50 USD. The README.md should indicate that this is possible, but was it not successful for you? If you tried and it didn't work, please specify the expected behavior and the actual behavior in your question.

**Comment by 1domybest** [comment]
Im really sorry didnt know that have to pay ㅠㅠ
Before sending the Rtmp video sample buffer, I was wondering if I could
modify it and send it as I wanted...? Thank you so much for your help

2024년 6월 20일 (목) 오후 9:27, shogo4405 ***@***.***>님이 작성:
